pkgs/tools/security/honggfuzz/default.nix

```diff
@@ -45,7 +45,7 @@ let
       homepage = "https://honggfuzz.dev/";
       license = lib.licenses.asl20;
       platforms = [ "x86_64-linux" ];
-      maintainers = with lib.maintainers; [ cpu ];
+      maintainers = with lib.maintainers; [ cpu chivay ];
     };
   };
 in honggfuzz
```

Comment:
Why do we have this let in here?

Reply:
Seems unnecessary to me. Shall it remove it now or create a separate PR?

Reply:
Agreed - I think it was just confusion on my part in the original commit.

I vote remove it as part of this PR but defer to anyone with merge privs if they feel otherwise :-P

Reply:
I'll just do it after the merge.